dumb-init.c

```diff
@@ -58,7 +58,9 @@ void reap_zombies(int signum) {
         DEBUG("A child with PID %d exited with exit status %d.\n", killed_pid, exit_status);
 
         if (killed_pid == child_pid) {
-            DEBUG("Child exited with status %d. Goodbye.\n", exit_status);
+            DEBUG("Child exited with status %d. Cleaning up its process group...\n", exit_status);
+            signal_handler(SIGTERM);
```

Comment:
Do you know if this is safe? There are some restrictions on what signal handlers can do:
https://www.gnu.org/software/libc/manual/html_node/Defining-Handlers.html#Defining-Handlers

I suspect it is since we're about to `exit()` anyway but just want to make sure we've thought about it. It would be pretty easy to combine the two signal handler functions to avoid having to make a function call if we wanted to.